docs/source_values.rst

```diff
@@ -178,12 +181,106 @@ In this example, the LED is lit only when the button is not pressed:
 
 Some tools combine the values of multiple sources:
 
-.. image:: images/combining_sources.*
+.. image:: images/source_values/combining_sources.*
     :align: center
 
 In this example, the LED is lit only if both buttons are pressed (like an
 `AND`_ gate):
 
+.. image:: images/source_values/combining_sources_led_buttons.*
+    :align: center
+
 .. literalinclude:: examples/combining_sources.py
 
 .. _AND: https://en.wikipedia.org/wiki/AND_gate
+
+While most devices have a :attr:`~Device.value` range between 0 and 1, some have
+a range between -1 and 1 (e.g. :class:`Motor`, :class:`Servo` and
+:class:`TonalBuzzer`). Some source tools output values between -1 and 1, which
+are ideal for these devices, for example passing :func:`~tools.sin_values` in:
+
+.. image:: images/source_values/sin_values.*
+    :align: center
+
+.. literalinclude:: examples/sin_values.py
+
+In this example, all three devices are following the `sine wave`_. The motor
```

Comment:
I wonder if it might be better to use cosine here instead of sine, as it might be easier to explain (in words) going from the highest value, down to the mid value, and then down to the lowest value; than it is to explain starting at the mid value, going up to the max value, then back to the mid value, then down to the lowest value ? :man_shrugging:

Reply:
I've added a note